src/structs/handle.rs

```diff
@@ -55,6 +55,24 @@ impl Handle {
         }
     }
 
+    pub fn to_string(&self) -> String {
+        // NOTE: its unclear why a distinction is needed for the
+        // prelude.
```

Comment:
Because you want to display top-level symboles as `X`, not as `<prelude>.X` (which is surprising to the user).